# HG changeset patch
# User Andreas Weber <octave@josoansi.de>
# Date 1546363988 -3600
#      Tue Jan 01 18:33:08 2019 +0100
# Branch stable
# Node ID df510676ca6a4ed96eb7940818f6a7592c56273d
# Parent  050ec530e6e4b2387a2a7a35fc08765d95f3336e
octave-dock-widget.cc: Fix copy & paste error. (bug #55347)

diff --git a/libgui/src/octave-dock-widget.cc b/libgui/src/octave-dock-widget.cc
--- a/libgui/src/octave-dock-widget.cc
+++ b/libgui/src/octave-dock-widget.cc
@@ -53,18 +53,18 @@ namespace octave
       {
         if (buttonlist.at (0)->x () < buttonlist.at (1)->x ())
           {
             m_default_float_button = buttonlist.at (0);
             m_default_close_button = buttonlist.at (1);
           }
         else
           {
-            m_default_float_button = buttonlist.at (0);
-            m_default_close_button = buttonlist.at (1);
+            m_default_float_button = buttonlist.at (1);
+            m_default_close_button = buttonlist.at (0);
           }
       }
 
     // the custom (extra) title bar of the widget
     m_title_widget = new QWidget ();
 
     m_dock_action = new QAction
       (QIcon (":/actions/icons/widget-undock.png"), "", this);
